packages/wonder-blocks-modal/components/two-column-modal.js

```diff
@@ -37,7 +37,7 @@ type BaseProps = {|
      * clicks too.)
      * @ignore
      */
-    onClickCloseButton?: () => void,
+    onClose?: () => void,
```

Comment:
Here too, also remove the optional.

Reply:
I ended up getting rid of the default prop since we pass this through to `ModalPanel` and if we define a default for this then it throws when we're using the `ModalLauncher` which defines a `closeModal` function on the `ModalContext` b/c we want to avoid situations where both are set.